app/src/main/java/net/skyscanner/backpack/demo/compose/ButtonsStory.kt

```diff
@@ -52,39 +52,26 @@ import net.skyscanner.backpack.compose.tokens.BpkDimension
 import net.skyscanner.backpack.compose.tokens.BpkSpacing
 import net.skyscanner.backpack.compose.tokens.LongArrowRight
 import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.components.ButtonComponent
+import net.skyscanner.backpack.demo.meta.ComposeStory
 import kotlin.time.Duration.Companion.seconds
 
 @Composable
-fun ButtonsStory(
-  size: BpkButtonSize,
-  modifier: Modifier = Modifier,
-) {
-  LazyColumn(
-    modifier
-      .fillMaxSize()
-      .padding(top = BpkDimension.Spacing.Md),
-  ) {
-    item {
-      ButtonsRow(
-        type = BpkButtonType.Primary,
-        size = size,
-        enabled = false,
-      )
-    }
-    items(BpkButtonType.values().filter { !it.linkType() }) {
-      ButtonsRow(
-        type = it,
-        size = size,
-        enabled = true,
-      )
-    }
-  }
-}
+@ButtonComponent
+@ComposeStory("Default")
+fun ButtonsDefaultSizeStory(modifier: Modifier = Modifier) =
+  ButtonsTypesDemo(BpkButtonSize.Default, modifier)
```

Comment:
Demo is a story, which just needs to have an overload with the specified parameter to be be used as a fully-functional story. ButtonsDefaultSizeStory is a story, ButtonsTypesDemo is a demo.